portfolio/src/main/webapp/script.js

```diff
@@ -11,18 +11,46 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+/**
```

Comment:
don't forget to clean up commented out code before submitting!

Reply:
Ok done!